examples/MultiLingual/article.md

```diff
@@ -0,0 +1,163 @@
+The GPU speeds up the Marqo machine learning model which processes our documents as we insert them. 
+These AWS machines are very easy to set up as 
+[SageMaker Jupyter Notebook instances](https://aws.amazon.com/pm/sagemaker/). 
+
+## The solution
+If we were to develop this on a traditional SQL database or search engine, we'd have to manually create a translation 
+layer to process the queries, or link each document with handcrafted translations. 
+
+An example of this would be to translate all the documents into English as they are stored. The search query would also 
+be translated into English, and a keyword search would be performed using a technology like Elasticsearch. However this
+is problematic as a translated sentence is a lossy approximation of the source language and it introduces a significant component (real-time translation) into the system. This results in poorer search relevancy, worse latency, and additional system complexity.  
+
+Tensor search, the technology that powers Marqo, outperforms traditional keyword search methods.
+
+First, we set up a Marqo instance on the machine, which has docker installed. Notice the `--gpus all` option. 
+This allows Marqo to use GPUs it finds on the machine. If the machine you are using doesn't have GPUs, then remove this option from the command. 
+```sh
+docker rm -f marqo; 
+docker run --name marqo -it -p 8882:8882 --gpus all --add-host host.docker.internal:host-gateway marqoai/marqo:latest
+```
+We use pip to install the Marqo client (`pip install marqo`) and the datasets python package (`pip install datasets`). 
+We will use the `datasets` package from [Hugging Face](https://huggingface.co/docs/datasets/index)
+to import the MultiEURLEX dataset.
+
+Then, we start work on our Python script. We start by loading the the validation splits for the English and Deutsch datasets:
+```python
+from datasets import load_dataset 
+dataset_en = load_dataset('multi_eurlex', 'en', split="validation")
+dataset_de = load_dataset('multi_eurlex', 'de', split="validation")
+```
+We then import Marqo and set up the client. We tell the Marqo client to connect with the Marqo Docker container that we ran earlier.
+
+```python
+from marqo import Client
+mq = Client("http://localhost:8882")    
+```
+Then, add a line telling Marqo to create the multilingual index: 
+```python
+mq.create_index(index_name='my-multilingual-index', model='stsb-xlm-r-multilingual')
+```
+Notice that here is where we tell Marqo what model to use. After this, we'll iterate through each dataset, indexing each document as we go. 
+One small adjustment we'll make is to split up text of very long documents (of over 100k chars) to make it easier to index and search. 
+At the end of each loop, we call the `add_documents()` function to insert the document:
+```python
+mq.index(index_name='my-multilingual-index').add_documents(
+    device='cuda', auto_refresh=False,
```

Comment:
auto_refresh is no longer an accepted parameter. We should remove this from all examples.